01week/rockPaperScissors.js

```diff
@@ -1,51 +1,67 @@
-'use strict';
+"use strict";
 
-const assert = require('assert');
-const readline = require('readline');
+const assert = require("assert");
+const readline = require("readline");
 const rl = readline.createInterface({
   input: process.stdin,
   output: process.stdout
 });
 
-
 function rockPaperScissors(hand1, hand2) {
```

Comment:
Your logic is correct but this will not pass the tests since it returns results that are different than one the tests want.